### DISCUSSION TITLE
Storing a list of items in an `_environment.local` file

### DISCUSSION THREAD

**Discussion by arnaudgallou** [quarto-dev/quarto-cli#10482] [Q&A] [answered]
### Description

The title says it all. Is it possible to store a list of items in an `_environment.local` file? If so, how?
I've a bunch of related Google sheets I'd like to read in a document. Instead of creating a variable for each file, I was thinking of storing the sheet IDs in a list, for convenience.
E.g.:

```
GS_KEYS=[
 "key_1",
 "key_2",
 "key_n"
]
```

This is to be handled in R.

**Comment by mcanouil** [comment, marked as answer]
`_environment` is following usual environment variable specification.
It does not have complex structure (most of the time depending on the environment being Bash, PowerShell, etc.), so use whatever separator and split afterwards.
